wcfsetup/setup/db/install.sql

```diff
@@ -1432,6 +1432,19 @@ CREATE TABLE wcf1_trophy(
 	KEY(categoryID)
 );
 
+DROP TABLE IF EXISTS wcf1_unfurl_url;
+CREATE TABLE wcf1_unfurl_url (
+	urlID INT(10) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	url TEXT NOT NULL,
+	urlHash VARCHAR(40) NOT NULL UNIQUE KEY,
+	title VARCHAR(255) NOT NULL DEFAULT '',
+	description TEXT, 
+	imageUrl TEXT, 
+	imageType ENUM('SQUARED', 'COVER', 'NOIMAGE') NOT NULL DEFAULT 'NOIMAGE',
+	imageHash VARCHAR(40) NOT NULL DEFAULT '',
+	status ENUM('PENDING', 'SUCCESSFUL', 'REJECTED') DEFAULT 'PENDING'
```

Comment:
I tend to avoid `ENUM`s in favor of a simple `VARCHAR`. This makes future adjustments easier.

Reply:
I personally dislike to have an `VARCHAR` there. With an `ENUM` you can guarantee, that there is an correct value, even if the user edits the table by himself (for whatever reason).

But I have changed it to an VARCHAR.